test/cluster.sh

```diff
@@ -111,6 +111,9 @@ function install_knative_serving() {
 
   wait_until_pods_running knative-serving || return 1
   wait_until_pods_running istio-system || return 1
+  if kubectl get -n istio-system knative-ingressgateway; then
```

Comment:
I would add a `> /dev/null 2>&1` here to avoid a confusing error message about a resource not found, when it's perfectly normal that it's not found.

Reply:
Done.